jsettlers.graphics/src/jsettlers/graphics/map/controls/original/panel/content/Label.java

```diff
@@ -20,12 +20,18 @@
 import jsettlers.graphics.utils.UIPanel;
 
 public class Label extends UIPanel {
+    public enum HorizontalAlignment {
+        LEFT,
+        CENTRE,
+        RIGHT,
+    }
 
 	private final EFontSize size;
-	private final String[] words;
+	private String[] words;
```

Comment:
I suppose I'm suggesting that the UI should have an event queue for events from the back end, which is probably a bit of a change for now, I've put something a bit simpler in.